Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -209,10 +209,17 @@ void RegisterModulesDynamically()
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory, ApiSettings apiSettings)
         {
             loggerFactory.AddLog4Net();
 
+            if (!string.IsNullOrEmpty(apiSettings.PathBase))
+            {
+                var pathBase = apiSettings.PathBase.Replace("/", "");
```

Comment:
Doing the slash handling this way is dangerous if they want to use a multipart path base such as "api/test".  This should only be normalizing trailing and leading slashes.